content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -77,6 +77,12 @@ when a cluster is under resource pressure. For this reason, it is not
 recommended to disable preemption.
 {{< /note >}}
 
+{{< note >}}
+In Kubernetes 1.15 and later,
+PriorityClasses have the option to set `preemptionPolicy: Never`.
+This will prevent pods of that PriorityClass from preempting other pods.
```

Comment:
I think we should point out that this feature is disabled by default. To do so, we can change this line to:
`If this feature is enabled, it will prevent ...`.